cypress/e2e/signIn.cy.js

```diff
@@ -18,21 +18,30 @@ describe('Sign In page', () => {
   });
 
   it('should provide an ability to log in with existing credentials', () => {
+    cy.register();
     signInPage.visit();
-    cy.register(user.email, user.username, user.password);
-
+    
     signInPage.emailField
-      .type(user.email);
+      .type('riot@qa.team');
     signInPage.passwordField
-      .type(user.password);
+      .type('12345Qwert!');
```

Comment:
Hardcoded password '12345Qwert!' used. It's recommended to set the password as a variable for better maintainability and security of the tests.